### ISSUE TITLE
Refactor cypress tests to playwright

### ISSUE THREAD

**Issue by mesudip** [IntersectMBO/govtool#733]
### 1. Wallet Connect
- [x] 1A. Should connect wallet and choose stake-key to use
- [x] 1B. Should connect wallet with single stake key
- [x] 1C. Should disconnect Wallet When connected
- [x] 1D. Should check correct network (Testnet/Mainnet) on connection

### 2. Delegation Tests
- [x] 2A. Should show delegated DRepId on dashboard after delegating
- [x] 2B. Should access delegation to dRep page
- [x] 2C. Verify AdaHolder Behavior in Disconnected State
- [x] 2D. Verify AdaHolder Behavior in Connected State
- [x] 2E. Should delegate to myself
- [x] 2F. Should change delegated DRep
- [x] 2G. Should verify retire as a DRep (Moved to 3I)
- [x] 2H. Verify DRep behavior in Retired State (Moved to 3J)
- [x] 2I. Should check validity of DRep Id

### 3. DRep Registration
- [x] 3A. Should show dRepId on dashboard after connecting registered dRep Wallet
- [x] 3B. Should access dRep registration page
- [x] 3C. Should open wallet connection popup, when Register as DRep from wallet unconnected state
- [x] 3D. Should access DRep registration page in Wallet Connected State
- [x] 3E. Should reject invalid data and accept valid data
- [x] 3F. Should create proper DRep registration request, when registered with data
- [x] 3G. Should show confirmation message with link to view transaction, when DRep registration txn is submitted
- [x] 3H. Should be able to update metadata
- [x]  3I. Should verify retired as DRep
- [x] 3J. Verify DRep behavior in retired state

### 4. Proposals Visibility
- [x] 4A. Should access Governance Actions page with/without connecting wallet
- [x] 4B. Should restrict voting for users who are not registered as DReps
- [x] 4C. Should filter Governance Action Type on governance actions page
- [x] 4D. Should sort Governance Action Type on governance actions page
- [x] 4E. Should filter and sort Governance Action Type on governance actions page

### 5. Proposals Functionality
- [x] 5A. Should show relevant details about governance action as dRep
- [x] 5B. Should view "Vote" button on governance action item on registered as DRep
- [x] 5C. Should show required field in proposal voting on registered as DRep
- [x] 5D. Should validate proposal voting
- [x] 5E. Should re-vote with new data on a already voted governance action
- [x] 5F. Should show notification of casted vote after vote
- [x] 5G. Should show warning to the users to visit the site at their own risk, when external url is opened
- [x] 5H. Should open a new tab, when external URL is opened
- [x] 5I. Should view the vote details,when viewing governance action already voted by the DRep

### 6. Miscellaneous
- [x] 6A. Should be accessible from mobile
- [x] 6B. Provides error for invalid format
- [x] 6C. Navigation within the dApp
- [x] 6D. Proper label and recognition of the testnet network

**Comment by NabinKawan** [comment]
It is completed. [PR](https://github.com/IntersectMBO/govtool/pull/939)

**Comment by NabinKawan** [comment]
6A. test has been moved into new ticket.
[https://github.com/orgs/IntersectMBO/projects/20/views/1?pane=issue&itemId=61865747](https://github.com/orgs/IntersectMBO/projects/20/views/1?pane=issue&itemId=61865747)

**Comment by NabinKawan** [comment]
Blocked by: #937  #938  #897

**Comment by Ryun1** [comment]
This ready to be closed now @spannercode ?
